### DISCUSSION TITLE
Using --nodes together with --time-request for job submissions

### DISCUSSION THREAD

**Discussion by Schlevidon** [It4innovations/hyperqueue#662] [Q&A]
Hi, I wanted to try out multi-node tasks, but I noticed that HQ doesn't allow combining the `--nodes` and `--time-request` arguments when submitting jobs. This functionality would be very useful for our application, so I was wondering if there are any plans to add this in the near future?

**Comment by spirali** [comment]
Hi,

It is on our roadmap, but we postponed it because we did not have a use case and there are some tricky parts that has to be solved. I will have some time next week and I will try to look on it.

**Reply by Schlevidon** [reply]
Sounds great, thanks a lot! :)

**Reply by spirali** [reply]
Just to let you know, I am now working on the feature. However, I need to finish some deeper changes in multinode scheduler so it will takes 1-2 weeks to finish all this.
